packages/devextreme/js/ui/text_area.js

```diff
@@ -205,7 +204,6 @@ const TextArea = TextBox.inherit({
         if(maxHeight !== undefined) {
             const adjustedMaxHeight = maxHeight - heightDifference;
             const needScroll = inputHeight > adjustedMaxHeight;
-
```

Comment:
Let's remove your changes with removing empty lines. I think that it doesn't matter in your current task.

Reply:
Correction: [Commit](https://github.com/DevExpress/DevExtreme/pull/26885/commits/8c3cd0d674579d15406941118b5f92d97ddecb40)